Move files to start tutorial

drhagen opened the pull request:
I did not get to make all the changes that I wanted to, but I moved the necessary files and also fixed them up enough so that they run. This should be a good starting point for people to check out and improve upon.

xpspectre commented:
Do we also want to merge to Models folder into the Tutorial folder? Or move models currently in the Tutorial and Testing folder into Models?

drhagen commented:
Yes, let's get rid of the `Models` folder. Let's keep only the models that pertain to the tutorial and put them that folder. I'll probably move all the models for now; we can trim down the selection later.

drhagen commented:
I moved all the models to the tutorial directory.

I also fixed up the existing tutorial scripts. This required fixing a bug with how to deal with simbio models having species like cell.C:C. I chose to allow cell."C:C", which also implements issue #70, allowing cell.species to exist without quotes. Try out different expressions and let me know if there should be tweaks to what is allowed. I am not 100% on board with cell."C:C" being favored above "cell.C:C".

xpspectre commented:
If I recall correctly, SimBio writes out components with invalid characters in their names using square brackets. For example `cell.C:C`, becomes `cell.[C:C]`. Is this right? This makes it easy to convert it to `cell."C:C"`, but I don't like it as much as `"cell.C:C"`. I know at one point, I wrote a bunch of code to do this the `"cell.C:C"` way, but I'm not sure it still works. I'm fine with how it works in this pull request for now, but will probably revisit issue #70 again shortly.

xpspectre commented:
I went ahead and merged this so I could get started on published docs.
